.github/workflows/documentation-update.yml

```diff
@@ -22,6 +22,12 @@ jobs:
       - name: Generate Docs
         run: yarn run docgen
         working-directory: packages/contracts/
+      - name: Remove repetitions from the generated contract docs introduced by solidity-docgen
+        run: find . -type f -name '*.md' -exec perl -i -0777 -pe 's/<!--CONTRACT_END-->.*//s' {} \;
+        working-directory: packages/contracts/docs/osx/03-reference-guide
+      - name: Format with prettier
+        run: yarn run prettier '**/*.md' --write
```

Comment:
@mathewmeconry Will this run only inside `packages/contracts/docs/osx/03-reference-guide`?